frame/staking/src/pallet/mod.rs

```diff
@@ -1083,7 +1098,7 @@ pub mod pallet {
 				suppressed: false,
 			};
 
-			Self::do_remove_validator(stash);
+			let _ = Self::try_remove_validator(stash);
```

Comment:
Why do we ignore returns value? (we obviously ignored it previously too, but should we? Or at least should we not document why we ignore it?)